slides/ruby-standard-library/index.html

```diff
@@ -0,0 +1,139 @@
+    <script>
+      var link = document.createElement( 'link' );
+      link.rel = 'stylesheet';
+      link.type = 'text/css';
+      link.href = window.location.search.match( /print-pdf/gi ) ? '../../css/print/pdf.css' : '../../css/print/paper.css';
+      document.getElementsByTagName( 'head' )[0].appendChild( link );
+    </script>
+  </head>
+  <body>
+    <div class="reveal">
+      <div class="slides">
+        <!-- Title slide with the topic. Don't forget to change the <title> tag too! -->
+        <section>
+          <div class="flex-container">
+            <div class="flex-column text-align-left">
+              <h1>Ruby Monstas</h1>
+              <hr>
+              <p class='subtitle'>
+                The Ruby Standard Library
+              </p>
+            </div>
+            <div class="flex-column-200px">
+              <img class="no-border" src="../images/logo-cropped-small.png">
+            </div>
+          </div>
+        </section>
+
+
+        <section>
+          <h2>The Ruby Distribution</h2>
+          <ul class="list-none list-reset">
+            <li class="border-4 border-red-500 p-4">
+              <ul class="list-none list-reset">
+                <li class="border-4 border-red-500 my-4 p-4 bg-yellow-500">
+                  <b>Ruby Standard Library</b>
+                  <div class="text-2xl">More advanced Classes/Features</div>
+                </li>
+                <li class="border-4 border-red-500 my-4 p-4">
+                  <b>Ruby Core</b>
+                  <div class="text-2xl">Strings, Symbols, Array, Hashes, ...</div>
+                </li>
+                <li class="border-4 border-red-500 my-4 p-4">
+                  <b>Ruby Language</b>
+                  <div class="text-2xl">Syntax</div>
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </section>
+
+        <section>
+          <h2>Contents / Purpose</h2>
+          <div class="fragment fade-in">
+            The <b>Ruby Standard Library</b> is an extensive set of advanced tools.
+          </div>
+          <div class="fragment fade-in my-8">
+            These tools provide functionalities for common problems.
+            <div class="text-2xl">We'll see examples later</div>
+          </div>
+          <div class="fragment fade-in">
+            The Library is <b>built-in</b>, so we don't have to install anything.
+          </div>
+        </section>
+
+        <section>
+          <h2>Examples</h2>
+          <ul class="list-reset">
+            <li class="fragment fade-in my-4">
+              <b>Networking</b>
+              <div class="text-2xl">Web, E-Mail, Sockets, ...</div>
+            </li>
+            <li class="fragment fade-in my-4">
+              <b>Advanced Math</b>
+              <div class="text-2xl">Prime number generator, Matrices, Complex numbers, ...</div>
+            </li>
+            <li class="fragment fade-in my-4">
+              <b>Date format libraries</b>
```

Comment:
```suggestion
              <b>Data format libraries</b>
```

I guess you meant that... ;)